docs/architecture/grpc-for-wcf-developers/authentication.md

```diff
@@ -1,11 +1,42 @@
 ---
-title: gRPC authentication and authorization - gRPC for WCF developers
-description: TO BE WRITTEN
+title: Security in gRPC applications - gRPC for WCF developers
+description: Overview of call and channel authentication and authorization in gRPC
 author: markrendle
 ms.date: 09/02/2019
 ---
 
-# gRPC authentication and authorization
+# Security in gRPC applications
+
+In any real-world scenario, securing applications and services is essential. Security covers two key areas: encrypting network traffic to prevent it from being intercepted by bad actors; and authenticating users to prevent unauthorized system access and apply permissions based on identity. This chapter will cover the facilities for authentication and authorization in gRPC for ASP.NET Core, and discuss network security using TLS encrypted connections.
```

Comment:
There are three areas of security. Authentication, Authorization and Encryption. See [this article](http://www.bu.edu/tech/about/security-resources/bestpractice/auth/) for an overview. WCF handles these three concepts separately so it might be worth mapping them to gRPC in these three categories.